docs/citing.rst

```diff
@@ -0,0 +1,40 @@
+.. include:: links.rst
+
+.. _citation:
+
+===============
+Citing FMRIPREP
+===============
+
+``fmriprep`` is a Nipype_-based pipeline that combines software from several
+neuroimaging packages with custom-written code to flexibly apply
+state-of-the-art preprocessing algorithms to a wide variety of structural and
+functional neuroimages and generate readable reports.
+For a full list of external software and relevant citations (where available)
+of tools and algorithms, please see the `REFERENCES.md
+<https://github.com/poldracklab/fmriprep/tree/master/RERFERENCES.md>`_ table in
+the source distribution.
```

Comment:
Can we add the picture of fmriprep poster somewhere in the docs, like I did with mriqc (http://mriqc.rtfd.io)?

https://f1000research.com/posters/6-1129

Reply:
Maybe in the citation page? Could have a sub-heading like:

```RST
Posters
----------
* OHBM 2017 (`pdf <https://f1000research.com/posters/6-1129>`_)
  [thumbnail linking to full-size PNG]
```

Reply:
Cool!

Reply:
Added poster in PNG format to docs/

Reply:
Had added one already. Just going to force push over, unless you're very attached to that png.

Reply:
not really :+1: